ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -393,6 +408,8 @@ open class NavigationBar: UINavigationBar, TwoLineTitleViewDelegate {
         rightBarButtonItemsStackView.setContentHuggingPriority(.defaultHigh, for: .horizontal)
         rightBarButtonItemsStackView.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
 
+        backButtonItem = UIBarButtonItem(image: UIImage.staticImageNamed("back-24x24"), style: .plain, target: nil, action: #selector(NavigationBarBackButtonDelegate.backButtonWasPressed))
```

Comment:
Do you need to set the accessibility label? I think right now VoiceOver will just read the image name, rather than a localized description.

Reply:
VoiceOver appears to read this by its title, so I think we're okay for now. It might be nicer if we could get VoiceOver to say "back button," but that's a separate can of worms.